source/extensions/tracers/opentelemetry/tracer.h

```diff
@@ -14,26 +14,19 @@
 
 #include "absl/strings/escaping.h"
 #include "span_context.h"
+#include "tracer_stats.h"
```

Comment:
please use full path

Reply:
Changed all in: [2a5d544](https://github.com/envoyproxy/envoy/pull/29207/commits/2a5d5448406e99fbfaecda651e0d176eda96383a)